AmericanMedicalAssociation/AMA-style-guide
# Ama 40 promo block primary cta atom

Note: Please review and merge https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/12 first, as this branch is based on it.

https://palantir.atlassian.net/browse/AMA-40

Adds the primary cta to the Promotional block. As the primary cta is a h6 header, I do not think that an individual atom is necessary.

### Steps to Test:

1. Navigate to Molecules > Blocks > Promotional Block.
2. Observe that there are three Promotional Blocks with primary ctas and titles.
